app/assets/javascripts/short-form/ShortFormDataService.js.coffee

```diff
@@ -468,6 +470,7 @@ ShortFormDataService = (ListingDataService, ListingConstantsService, ListingPref
         unless prefKey
           # must be a customPreference... just identify by ID much like on e7b-custom-preferences
           prefKey = listingPref.listingPreferenceID
+          isCustom = true
```

Comment:
this feels a bit hacky, but keeps adding the inidividualPreference piece limited to custom preferences.